src/pageObjects/welcomePage/components/SignUpPopUp.ts

```diff
@@ -19,31 +19,26 @@ export class RegistrationPage {
 
 
     public constructor(page: Page) {
+        super(page, page.locator('app-signup-modal'));
         this.page = page;
 
-        this.btnSignUp = page.locator('button.hero-descriptor_btn')
-        this.btnRegister = page.locator('div.modal-footer button')
-        this.btnRemoveMyAccount = page.locator('.close')
+        this.btnRegister = this.container.locator('div.modal-footer button')
+        this.btnRemoveMyAccount = this.container.locator('.close')
 
-        this.inputName = page.locator('#signupName')
-        this.inputLastName = page.locator('#signupLastName')
-        this.inputEmail = page.locator('#signupEmail')
-        this.inputPassword = page.locator('#signupPassword')
-        this.inputReEnterPassword = page.locator('#signupRepeatPassword')
+        this.inputName = this.container.locator('#signupName')
+        this.inputLastName = this.container.locator('#signupLastName')
+        this.inputEmail = this.container.locator('#signupEmail')
+        this.inputPassword = this.container.locator('#signupPassword')
+        this.inputReEnterPassword = this.container.locator('#signupRepeatPassword')
 
-        this.invalidNameMsg = page.locator('div.invalid-feedback p')
-        this.signUpForm = page.locator('.modal-content')
-    }
-
-    async clickBtnSignUp() {
-        await this.btnSignUp.click()
+        this.invalidNameMsg = this.container.locator('div.invalid-feedback p')
```

Comment:
you use with name, lastName and email, it looks inconsistent

we already talked about that, connect error message locator to input locator